onnxruntime/test/framework/insert_cast_transformer_test.cc

```diff
@@ -134,5 +134,24 @@ TEST(TransformerTest, ThreeInARowRemoval) {
   ASSERT_TRUE(op_to_count["Cast"] == 2);
 }
 
+// test a case where the ONNX inferred type (float16) is different from the type bound
+// to the output NodeArg of the "EyeLike" node because of the InsertCaseTransformer
+TEST(TransformerTest, EyeLikeModelWithFloat16Inputs) {
```

Comment:
nit: RandomNormalLikeWithFloat16Inputs?

Reply:
Fixed. Initially started of with EyeiLike and switched to RandomNormalLike afterwards and forgot to update the test name.